prime-router/src/main/kotlin/metadata/LivdLookup.kt

```diff
@@ -0,0 +1,206 @@
+package gov.cdc.prime.router.metadata
+
+import gov.cdc.prime.router.Metadata
+
+// this will be used by the LivdMappers and the CustomFHIRFunction
+// /need to move most of LivdMappers in here
+// will need to move the tests to look at this as well
+// keep the library the way it was then use the context to use the library to pass in custom functions
+object LivdLookup {
+    fun find(
+        testPerformedCode: String?,
+        processingModeCode: String?,
+        deviceId: String?,
+        equipmentModelId: String?,
+        testKitNameId: String?,
+        equipmentModelName: String?,
+        tableColumn: String,
+        tableRef: LookupTable? = Metadata.getInstance().findLookupTable(name = "LIVD-SARS-CoV-2")
+    ): String? {
+        if (tableRef == null) {
```

Comment:
This is really an error and not a null return. Null means a lookup was really done, but no value was found